src/components/Trade/Components/TradeQuotes/TradeQuotes.tsx

```diff
@@ -18,11 +21,12 @@ type TradeQuotesProps = {
 }
 
 export const TradeQuotes: React.FC<TradeQuotesProps> = ({ isOpen, isLoading }) => {
-  const buyAssetFiatRate = useSwapperStore(selectBuyAssetFiatRate)
+  const buyAssetUsdRate = useSwapperStore(selectBuyAssetUsdRate)
```

Comment:
pretty sure this is a production bug. we should not be using fiat rates to convert USD amounts to to crypto